README.md

```diff
@@ -21,7 +21,9 @@ Generally, the order book looks like the following example (B - bid, S - spread,
 To build and compile the project, follow these steps:
 - Open a terminal and navigate to the root directory of the project:`cd TestTaskProxyBand`
 - Run the following command to build the JAR file: `mvn clean package`
-- Then start the program with the following command: `java -jar <your-project>.jar`
+
+## Usage
+- To start the program with the following command: `java -jar <your-project>.jar`
```

Comment:
Remove the list here, because it consists of only 1 element

Reply:
fix